libcxx/include/__chrono/time_zone_types.h

```diff
@@ -0,0 +1,112 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// For information see https://libcxx.llvm.org/DesignDocs/TimeZone.html
+
+#ifndef _LIBCPP___CHRONO_TIME_ZONE_TYPES_H
```

Comment:
As discussed during live review, maybe this whole header can be an implementation detail of the dylib if we use pimpl more?

Reply:
I've done this. It would be possible to add proper constructors to these classes instead of builder functions in the dylib. I want to do that at a later stage, just in case parts are needed in the header. (I don't expect that but the code still misses a lot of functionality.)

A disadvantage of moving the code to the dylib is that testing it is harder. At the moment it's quite impossible since the code that uses the data has not been implemented. For now I disabled the tests and left TODOs so I can revisit that later.